src/components/dev-dot-content/mdx-components/mdx-headings/mdx-headings.module.css

```diff
@@ -27,6 +28,16 @@
 		font-size: 0.8em;
 		font-weight: 500;
 	}
+
+	&:hover {
+		& :global(.__permalink-h) {
+			opacity: 1;
+
+			@media (prefers-reduced-motion: no-preference) {
+				transition: opacity 0.2s ease;
+			}
```

Comment:
Is this needed here since it's declared up in `MdxHeadingPermalink`'s `.root` class?

Reply:
Nope. Good call!